project-base/storefront/components/Pages/Customer/EditProfile/EditProfileContent.tsx

```diff
@@ -81,49 +73,6 @@ export const EditProfileContent: FC<EditProfileContentProps> = ({ currentCustome
         return () => handleFormErrors(changeProfileResult.error, formProviderMethods, t, formMeta.messages.error);
```

Comment:
This arrow function could be removed as we are no longer postponing the error/success messages.

Reply:
fixed